website/project/model.py

```diff
@@ -4588,3 +4622,36 @@ def approve(self, user):
     def reject(self, user):
         self.approval.reject(user)
         self.approval.save()
+
+
+@unique_on(['destination_node', '_id'])
+class MailingListEventLog(StoredObject):
+    _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
+
+    date_created = fields.DateTimeField(default=datetime.datetime.utcnow, index=True)
+
+    content = fields.StringField()
+    sending_email = fields.StringField(index=True)
+    sending_user = fields.ForeignField('user', index=True)
+    destination_node = fields.ForeignField('node', index=True)
+    status = fields.StringField()
```

Comment:
That would be determined by the contributors on `destination_node`, but I suppose that it's worth logging because contributors can change.